sys/include/net/gnrc/netdev2.h

```diff
@@ -73,6 +73,16 @@ typedef struct gnrc_netdev2 {
     gnrc_pktsnip_t * (*recv)(struct gnrc_netdev2 *dev);
 
     /**
+     * @brief Set the state of this device
+     */
+    int (*_set_netdev_state)(struct gnrc_netdev2 *dev, netopt_state_t devstate);
```

Comment:
Opps, I think you are right. But, my original idea is to provide more specific MAC operation functions, like

`gnrc_mac_set_state(gnrc_netdev2, NETOPT_STATE_IDLE)`
`gnrc_mac_get_state(gnrc_netdev2)`
`gnrc_mac_enable_csma(gnrc_netdev2, true)`
`gnrc_mac_enable_autoack(gnrc_netdev2, true)`

To make it look more straightforward for MAC developers to use. I am not sure whether is it a good idea now.. Or, should I move these specific functions into my "gnrc_mac" module